### DISCUSSION TITLE
Copy button on review comment about file/line

### DISCUSSION THREAD

**Discussion by adamisproductiv** [community/community#7745] [Pull Requests]
I'd like a copy filename button button on the code review panes file name header in the PR discussion screen so I can easily copy the file to get to it in my editor.

**Comment by WillsB3** [comment]
I'd also like this. Currently if you try and use the browsers normal text selection features to manually copy the file path the pane content expands/collapses on `mouseup` which can be confusing and lead to missing further comments in that file.

**Comment by dzhavat** [comment]
Just came to propose the same feature. I really miss it when reviewing PRs. Got a screenshot as well. The top part is the comment in the "Conversation" page, the bottom part is the same file in "Files changed" page.

![image](https://user-images.githubusercontent.com/1096332/149224444-8eff5c77-dcdf-4b82-b17a-baa4c478be60.png)

**Comment by sousajunior** [comment]
It's looks like so easy to implement and the gain with this will be giant.
I've been waiting for this since I started using github every day.

**Comment by etsvetanov** [comment]
I've given up on using Github UI for doing PR reviews - loading a big PR is reaaaaaly slow. Simple things missing, like the issue with collapsing comment when you just try to copy the file path... just use vscode github extension

**Comment by cizmarf** [comment]
I just created a Chrome extension covering this feature. Hope it will work for some time and Github will not make any changes on their ui. https://github.com/cizmarf/github-pr-file-copy
